compact/far_forward.xml

```diff
@@ -9,7 +9,9 @@
   <include ref="far_forward/B0_tracker.xml"/>
   <include ref="far_forward/B0_ECal.xml"/>
   <include ref="far_forward/offM_tracker.xml"/>
-  <include ref="far_forward/ZDC.xml"/>
+  <!--  <include ref="far_forward/ZDC.xml"/> -->
```

Comment:
Please remove commented lines.

Reply:
this pull request was already approved, so I created a new pull request with the commenting-out of that line